docs/api/app.md

```diff
@@ -676,6 +676,54 @@ Overrides the current application's name.
 
 **Note:** This function overrides the name used internally by Electron; it does not affect the name that the OS uses.
 
+### `app.setNodePreload(filepath)`
+
+* `filepath` string | null
+
+Sets a preload script that will be loaded by all Node.js environments in this
+app.
```

Comment:
```suggestion
Sets a preload script that will be run before a Node.js environment is initialized.
```

"all Node.js environments" is a bit confusing, and it's expanded on below, so let's drop that part.